builder/ConfTargets.cmake

```diff
@@ -76,6 +76,14 @@ endif()
 
 option( MFX_ENABLE_KERNELS "Build with advanced media kernels support?" ON )
 
+if ( ${version} STRGREATER "1025" )
```

Comment:
Please, use ${API_VERSION} here instead of ${version}. And use standard version check in cmake like VERSION_GREATER. Mind that not all version checks are supported in cmake-2.8 which we follow now. I.e. your check should look like:
if (${API_VERSION} VERSION_GREATER 1.24)